set.mm

```diff
@@ -739364,6 +739491,21 @@ those for the more general case of a piecewise smooth function (see
       cvv eqeltrd ) ABCBFGZHZSATBACBIDEJKASABCSQDABCHQACIDLEMSNOPR $.
   $}
 
+  ${
+    $d E x y $.  $d S x y $.  $d ph x y $.
+    salrestss.1 $e |- ( ph -> S e. SAlg ) $.
+    salrestss.2 $e |- ( ph -> E e. S ) $.
+    $( A sigma-algebra restricted to one of its elements is a subset of the
+       original sigma-algebra (Contributed by Glauco Siliprandi,
```

Comment:
@glacode : when you have time: there are missing periods at the end of this and three other comments in this PR.

Reply:
PR #4516 should have fixed these